changelog.txt

```diff
@@ -1,3 +1,12 @@
+0.1.4-preview
+============
+New Features:
+**Microsoft Identity Web provides an option to specify if the token acquision service should be a singleton**. See [issue for more details](https://github.com/AzureAD/microsoft-identity-web/issues/1).
```

Comment:
```suggestion
**Microsoft Identity Web provides an option to specify if the token acquisition service should be a singleton.** See [issue for more details](https://github.com/AzureAD/microsoft-identity-web/issues/1).
```

Reply:
leaving the "**." as that's how it is in the other change logs. thx for the other edits.